src/Proto.Actor/Mailbox/Mailbox.cs

```diff
@@ -85,14 +85,41 @@ IMailboxQueue userMailbox
 
         public void PostUserMessage(object msg)
         {
-            _userMailbox.Push(msg);
-
-            foreach (var t in _stats)
+            // if the message is a batch message, we unpack the content as individual messages in the mailbox
+            // feature Aka: Samkuvertering in Swedish...
+            if (msg is IMessageBatch  || msg is MessageEnvelope e && e.Message is IMessageBatch)
```

Comment:
New feature, IMessageBatch. an interface that if present, the mailbox will unwrap the contents as individual messages and place on the mailbox.
This way, we can send what looks like one single message, with sender and headers etc etc.
But the mailbox on the receiver side can consume and apply supervision to each individual message in the batch